### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by msyyc**
# Cadl Ranch Contribution Checklist:

- [ ] I have written a [scenario spec](../docs/writing-scenario-spec.md)
- [ ] I have **meaningful** `@scenario` names. Someone can look at the list of scenarios and understand what I'm covering.
- [ ] I have written a [mock API](../docs/writing-mock-apis.md)
- [ ] I have used `@scenarioDoc`s for extra scenario description and to tell people **how to pass** my mock api check.

### CODE REVIEW

File: packages/cadl-ranch-specs/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure-tools/cadl-ranch-specs",
-  "version": "0.16.2",
+  "version": "0.16.3",
```
Comment (weidongxu-microsoft) [inline packages/cadl-ranch-specs/package.json:3]: I am not sure why specs is patch update (0.16.2 to 0.16.3), while cadl-ranch-expect and cadl-ranch is minor update?

Could you have Chenjie for double check?

Comment (msyyc) [reply packages/cadl-ranch-specs/package.json]: With reference of last time, I find all version is bumped to `0.X+1.0`. So I think 0.17.0 is better. add @tadelesh to confirm

Comment (tadelesh) [reply packages/cadl-ranch-specs/package.json]: 0.17.0 is better for tsp upgrade.

Comment (msyyc) [reply packages/cadl-ranch-specs/package.json]: done

**Review by tadelesh — APPROVED** [review]

**Review by timotheeguerin — APPROVED** [review]

**Pull request merged**